src/Maestro/maestro-angular/src/app/model/build-status.ts

```diff
@@ -1,15 +1,30 @@
 // Result format of Azure DevOps api
 // https://docs.microsoft.com/en-us/rest/api/azure/devops/build/builds/list?view=azure-devops-rest-5.0
 
-export interface BuildListResult {
+export interface BuildListResultCompleted {
   count: number;
-  value: BuildStatus[];
+  value: BuildStatusCompleted[];
 }
 
-export interface BuildStatus {
+export interface BuildStatusCompleted {
   id: number;
   finishTime: string;
-  status: "all" | "cancelling" | "completed" | "inProgress" | "none" | "notStarted" | "postponed";
+  status: "completed";
   result: "canceled" | "failed" | "none" | "partiallySucceeded" | "succeeded";
   length: number;
 }
+
+export interface BuildListResultInProgress{
+  count: number;
+  value: BuildStatusInProgress[];
+}
+
+export interface BuildStatusInProgress {
+  id: number;
+  status: "inProgress";
+  _links: {
```

Comment:
By this change completed and in-progress has different way how to build web URL. I would recommend to either stay consistent with former implementation `this.getBuildLinkFromAzdo(build.azureDevOpsAccount as string, build.azureDevOpsProject as string, recentFailure.id);`  or modify the code for **completed** so it consistently gets URI from _links in both cases.